recipes/travis-encrypt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: travis-encrypt
+  version: "0.6.0"
+
+source:
+  git_url: https://github.com/mandeep/Travis-Encrypt.git
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - travis-encrypt=travis.cli:cli
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - click >=6.6
+    - cryptography >=1.7.1
+    - pyyaml >=3.12
+    - requests >=2.12.5
+
+  run:
+    - python
+    - setuptools
+    - click >=6.6
+    - cryptography >=1.7.1
+    - pyyaml >=3.12
+    - requests >=2.12.5
+
+test:
+  imports:
+    - travis
+    - travis.tests
+
+  commands:
+    - travis-encrypt --help
+
+about:
+  home: https://www.github.com/mandeep/Travis-Encrypt
+  license: GNU General Public License v3 or later (GPLv3+)
```

Comment:
`license: GPL-3.0`

Also for GPL, we typically require a `license_file` to be specified. See:

https://github.com/mandeep/Travis-Encrypt/issues/4

https://conda-forge.org/docs/meta.html#packaging-the-license-manually